src/confcom/azext_confcom/kata_proxy.py

```diff
@@ -129,9 +129,10 @@ def kata_genpolicy(
 
         if settings_file_name:
             arg_list.append("-j")
-            # only take the last part of the path for the settings file
-            settings_file_name = os.path.basename(settings_file_name)
             arg_list.append(settings_file_name)
+        else:
+            arg_list.append("-j")
```

Comment:
Since it looks like you're adding -j to the arg list regardless, put it before the if/else